### PR TITLE
Progress Bar Tests

### PR DESCRIPTION
**Pull request by AFogarty72**

### CODE REVIEW

File: test/progress-bar-test.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from 'chai';
+
+import { ProgressBar } from '../src';
+
+describe('<ProgressBar/>', function() {
+  it('Should have a default class of primary ithout type property', function() {
```
Comment (jwdotjs) [inline test/progress-bar-test.js]: without (spelling)

File: test/progress-bar-test.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    const component = shallow(<ProgressBar percentage={0} type='danger'/>);
+    expect(component.find('div.progress > div').hasClass('progress-bar-danger')).to.equal(true);
+  });
+
+  it('By default should not be animated', function() {
+    const component = shallow(<ProgressBar percentage={0}/>);
+    expect(component.find('div.progress > div').hasClass('animated')).to.equal(false);
+  });
+
+  it('Should have a default label of empty string', function() {
```
Comment (jwdotjs) [inline test/progress-bar-test.js]: Maybe we should change the text to something like `Should not display a label`, currently the test sounds like it should show a label html tag but have nothing inside it

**Comment by jwdotjs** [comment]
Everything looks good pending feedbac!

**Comment by AFogarty72** [comment]
Thanks I'll fix those things up today!!

**Pull request merged**